library/src/main/java/org/mustangproject/Item.java

```diff
@@ -310,22 +314,34 @@ public Item(NodeList itemChilds, boolean recalcPrice) {
 				addAdditionalReference(rdoc);
 			}
 		}
-		addReferencedLineID( referencedLineID );
+		addBuyerOrderReferencedDocumentLineID( buyerOrderReferencedDocumentLineID );
+		addBuyerOrderReferencedDocumentID( buyerOrderReferencedDocumentID );
 	}
 
 
+	public Item addBuyerOrderReferencedDocumentLineID(String s) {
+		buyerOrderReferencedDocumentLineID = s;
+		return this;
+	}
+
+	@Deprecated(since = "2.14.0")
 	public Item addReferencedLineID(String s) {
-		referencedLineID = s;
+		return addBuyerOrderReferencedDocumentLineID(s);
+	}
```

Comment:
I renamed this method to align it with the getter (which is part of the interface) and also be more consistent. I have kept and deprecated the old method for backward compatibility.